modules/@apostrophecms/schema/ui/apos/mixins/AposInputMixin.js

```diff
@@ -23,9 +23,19 @@ export default {
     triggerValidation: {
       type: Boolean,
       default: false
+    },
```

Comment:
This does not make sense, we are not creating a new piece every time we change an attachment field. It is just one more piece of data that lives contained under its own field name (inside some document, but that is not the component's concern).